src/odemis/util/transform.py

```diff
@@ -431,6 +431,30 @@ def __call__(self, x):
         return self.apply(x)
     __call__.__doc__ = apply.__doc__
 
+    def fre(self, x, y):
+        """
+        Returns the RMS value of the fiducial error registration (FRE).
```

Comment:
Why use FRE and not the target registration error? If I google FRE, I get multiple results saying to use TRE.

> Fiducial Registration Error (FRE) is often used as a surrogate for the Target Registration Error (TRE) that is the quantity of clinical interest [4]. Unfortunately, it has been shown [5] that TRE and FRE are uncorrelated. FRE is thus an unreliable predictor of registration accuracy. (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4423533/)

Could you also add another sentence briefly explaining when to use this function. Something like:
>This returns a measure for the registration error between source and target frame.

Reply:
I updated the docstring to clarify